public/js/typology-list.ts

```diff
@@ -0,0 +1,55 @@
+import { List } from './list.class';
+declare global { var list: List }
+
+list = new List();
```

Comment:
`const list = new List()`

Reply:
List has no type declaration in line 4 because has been declared as global var in line 2. ¿Is this incorrect?  
But thinking about, it is not necessary to declare this var as global. Removing line 2 and adding const to list in line 4 will work perfectly.